README.md

```diff
@@ -362,18 +359,18 @@ we are getting this output:
+| Typed Classes                             | `let x: Array<foo>;`<br />`let x: Class<foo>;`<br />`let x: MyClass<bar>;`                                                                                              | `{ "name": "<type>", "elements": [{ <element-type> }, ...] }`                                                                                                                                                                                                      |
+| Object Signature                          | `let x: { foo: string, bar?: mixed };`<br />`let x: { [key: string]: string, foo: number };`                                                                            | `{ "name": "signature", "type": "object", "raw": "<raw-signature>", "signature": { "properties": [{ "key": "<property-name>"|{ <property-key-type> }, "value": { <property-type>, "required": <true/false> } }, ...] } }`                                          |
+| Function Signature                        | `let x: (x: string) => void;`                                                                                                                                           | `{ "name": "signature", "type": "function", "raw": "<raw-signature>", "signature": { "arguments": [{ "name": "<argument-name>", "type": { <argument-type> } }, ...], "return": { <return-type> } } }`                                                              |
+| Callable-Object/Function-Object Signature | `let x: { (x: string): void, prop: string };`                                                                                                                           | `{ "name": "signature", "type": "object", "raw": "<raw-signature>", "signature": { "properties": [{ "key": "<property-name>"|{ <property-key-type> }, "value": { <property-type>, "required": <true/false> } }, ...], "constructor": { <function-signature> } } }` |
+| Tuple                                     | `let x: [foo, "value", number];`                                                                                                                                        | `{ "name": "tuple", "raw": "<raw-signature>", "elements": [{ <element-type> }, ...] }`                                                                                                                                                                             |
+| Union                                     | `let x: number | string;`                                                                                                                                               | `{ "name": "union", "raw": "<raw-signature>", "elements": [{ <element-type> }, ...] }`                                                                                                                                                                             |
```

Comment:
Can we escape the pipe in the union somehow or use an escape sequence?

It is currently treated as table separator.

Reply:
bah sorry not sure what's happening thing, maybe prettier being over zealous...i'll fix